cdap-docs/tracker-manual/source/index.rst

```diff
@@ -325,6 +344,57 @@ specified, in addition to common time ranges (two weeks to one year).
   :align: center
   :class: bordered-image
 
+**Preview**
+
+The *Preview* tab shows a preview for the dataset. It is available for all datasets that are
+explorable. You can scroll for up to 500 records. For additional analysis, you can leverage the
+*Jump* menu to go into CDAP and explore the dataset using a custom query.
+
+.. figure:: /_images/tracker-preview.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+**Usage**
+
+The *Usage* tab shows a set of graphs displaying usage metrics for the dataset. At the top is a
+histogram of all audit messages for a particular dataset. Along the bottom of the screen is a set of
+charts displaying the Applications and Programs that are accessing the dataset, and a table showing
+the last time a specific message was received about the dataset. Clicking the Application name or
+the Program name will take you to that entity in the main CDAP UI.
+
+.. figure:: /_images/tracker-usage.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+**Preferred Tags**
+
+The *Tags* tab at the top of the page allows you to enter a common set of preferred terms to use when
+adding tags to datasets. Preferred tags show up first when adding tags, and will guide your team to
+use the same terminology. Any Preferred Tag that has not been attached to any datasets can be deleted
+by clicking the red trashcan icon. If a Preferred tag ahs been added to an entity, you cannot delete it,
+but you can demote it back to just being a user tag.
+
+.. figure:: /_images/tracker-tags.png
+  :figwidth: 100%
+  :width: 800px
+  :align: center
+  :class: bordered-image
+
+Adding preferred tags is easy. Just click the **Add Preferred Tags** button and use the UI to add a
+list of terms you would like to be preferred. If the tag already exists in CDAP, it will be promoted
+to a preferred tag automatically. If it is a new tag in CDAP, it will still show up in the Preferred
+Tags list.
```

Comment:
```
To add preferred tags, click the **Add Preferred Tags** button and use the UI to add or import a
list of tags that you would like to be "preferred". If the tag already exists in CDAP, 
it will be promoted from being a user tag to being a preferred tag. If it is a new tag
in CDAP, it will be added in the *Preferred Tags* list.